.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+# You can find the deployment instructions in the scripts/cloud-api-demo/README.md file
+name: Deploy to Cloud VPS
+
+on:
+  push:
+    branches:
+      - develop
+      - deployment
```

Comment:
Do we have this branch? This should be removed

Reply:
Yes I was to remove that after some testing, I just finished all the test and added the final commit. Removing it

Reply:
Removed it